_posts/2019-01-21-ion-hive-serde.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: news_item
+title: "Ion Hive SerDe 0.1.0 Released"
+date: 2019-01-21
+categories: news
+---
+Apache Hive SerDe (short for serializer/deserializer) for the Ion file format alpha is released.
+
+This SerDe supports both Ion's binary and text format and all Ion types and flattening
+through [path extraction](https://github.com/amzn/ion-java-path-extraction). See the repository
+for a complete list of features.
```

Comment:
To be more consistent with the other news items, consider:


This new **Ser**ializer/**De**serializer for Apache Hive supports the Ion binary and text formats, all of the Ion types, and flattening through [path extraction](https://github.com/amzn/ion-java-path-extraction). This release is an Alpha release and a complete list of features can be found in the GitHub repository.